tox.ini

```diff
@@ -16,10 +16,7 @@ deps =
     ; Django versions
     django111: Django>=1.11,<1.12
     django22: Django>=2.2,<2.3
-    ; Python versions
-    py{35,36,37,38}: django-perf-rec
```

Comment:
I don't get why we need to remove this line

Reply:
it's in the commit comment: https://github.com/peopledoc/django-formidable/pull/386/commits/19fe8a69488b8d7883c3d7bd9fc54ef1fac55c2e